src/style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --link-color: #00acdc;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+/* HEADER */
+
+.header__wrapper {
+  width: 100%;
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  margin: 12px 0 10px;
+  width: 40px;
+  height: 40px;
+}
+
+.header__nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav__link:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link_hovered:hover,
+.is-active {
+  color: var(--link-color);
+  position: relative;
+}
+
+.nav__link_active:after {

```

Comment:
class `is-active` must have blue line.

Reply:
@gestap 'link with blue color and line below is an active link. It should have class="is-active" and relevant styles'. Link has this class, and relevant class for this line. Or what do you mean when write 'must'?

Reply:
@gestap I fixed it. Now class `is-active` is only for tests

Reply:
1. class `is-active` must be in the project
2. elements with class `is-active` must have `color: #00acdc;` and underline.

if we want another item in the navigation was blue and with an underline, we need only add to him class `is-active`.

If you don't understand what I mean, write to me in slack.